app/templates/community/meetups.hbs

```diff
@@ -1 +1,43 @@
-{{outlet}}
\ No newline at end of file
+<section aria-labelledby="Meetups">
+  <h1>Meetups Around the World</h1>
+
+  <h2>Find a meetup in your city</h2>
+
+  {{!-- Google Maps is yet to be done --}}
```

Comment:
I think we should try to implement the google maps as part of this PR. Do you want to take a look at it?